exercises/concept/regular-chatbot/.meta/exemplar.js

```diff
@@ -0,0 +1,65 @@
+// @ts-check
+
+/**
+ * Given a certain command, help the chatbot recognize whether the command is valid or not.
+ *
+ * @param {string} command
+ * @returns {boolean} whether or not is the command valid
+ */
+
+export function isValidCommand(command) {
+  const regex = /^chatbot/i;
+  return regex.test(command);
+}
+
+/**
+ * Given a certain message, help the chatbot get rid of all the emoji's encryption throught the message.
+ *
+ * @param {string} message
+ * @returns {string} The message without the emojis encryption
+ */
+export function removeEmoji(message) {
+  const regex = new RegExp('emoji[0-9]+');
+  return message.split(regex).join('');
+}
+
+/**
+ * Given a certain phone number, help the chatbot recognize whether it is in the correct format.
+ *
+ * @param {string} number
+ * @returns {string} the Chatbot response to the phone Validation
+ */
+export function checkPhoneNumber(number) {
+  const regex =
+    /^\(\+?([0-9]{2})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{3})[-. ]?([0-9]{3})$/;
+  if (regex.test(number)) {
+    return 'Thanks! You can download me now on your Phone.';
```

Comment:
```suggestion
    return 'Thanks! You can now download me now to you phone.';
```
If you accept this suggestion, make sure the instruction file is consistent.

Reply:
Great! Thanks for reminding to change it on the instructions file. I'll implement it in both files.

Reply:
I just saw my suggestion was wrong. It should say "Thanks! You can now download me to your phone." Maybe you already noticed that.